gensim/test/test_fasttext.py

```diff
@@ -1300,6 +1306,253 @@ def _run(self, fin):
         self.assertTrue(np.allclose(_ARRAY, array))
 
 
+def _create_and_save_test_model(fname, model_params):
+    model = FT_gensim(**model_params)
+    lee_data = LineSentence(datapath('lee_background.cor'))
+    model.build_vocab(lee_data)
+    model.train(lee_data, total_examples=model.corpus_count, epochs=model.epochs)
+    gensim.models.fasttext.save_facebook_model(model, fname)
+    return model
+
+
+def calc_max_diff(v1, v2):
+    return np.max(np.abs(v1 - v2))
+
+
+class SaveFacebookFormatModelTest(unittest.TestCase):
+    """
+    This class containts tests that check the following scenario:
+
+    + create fastText model1
```

Comment:
This comment describes _what_ the code does. It isn't really helpful, because that's already obvious from the code itself.

If it's not obvious, then the source code needs to be clearer. In this particular case, I think the code is OK, so you just need to get rid of the comment.

Reply:
Corrected.